src/Arcus.Templates.WebApi/appsettings.json

```diff
@@ -3,6 +3,7 @@
   "CertificateSubject": "<your-certificate-subject-name>",
   //#endif
   //#if (Serilog)
```

Comment:
What exactly does this `Serilog` conditional define ?
Do we have an option to setup logging without using Arcus.Observability & Serilog ?
If we do, then I think we need to use another conditional to determine if the APPINSIGHTS_INSTRUMENTATIONKEY should be included or not.
If we don't, then I think it makes sense to rename that conditional (in another PR).

Reply:
Yes, that's correct. So we have an logging project option that lets the user decide whether to use the default Console or the Serilog/Arcus AzureApplicationInsights setup. The `APPINSIGHTS_INSTRUMENTATIONKEY` is therefore only used with this second option and can therefore also be placed in the same conditional expression.

Reply:
Then I think the SErilog conditional should be renamed for clarity :)  (Arcus_AppInsights ? Serilog_AppInsights ? AppInsightsLogging ?)

Reply:
We could maybe discus this in another issue, and not burden Vincent with this 😄 .
We could indeed add some relationship with AppInsights in the directive, but `Serilog` should be part of that, as the project option is adding Serilog to the project. 😉

Reply:
Hm. Out of those three, `Serilog_AppInsights` makes the most sense to me.
I assume this refers to the same conditional?:

```
#if Serilog
using Serilog;
using Serilog.Configuration;
using Serilog.Events; 
#endif
```

Those usings are very Serilog specific so I'd expect some reference to Serilog to be in there.

Reply:
> We could maybe discus this in another issue, and not burden Vincent with this 😄 . We could indeed add some relationship with AppInsights in the directive, but `Serilog` should be part of that, as the project option is adding Serilog to the project. 😉

Yes, that's why I also mentionned that we should do it via another PR :)
I'll create an issue for this.